client/src/components/Navbar/Navbar.js

```diff
@@ -1,21 +1,102 @@
+import Icon from "../Icon/Icon";
+import PropTypes from "prop-types";
 
 // Function based React Component
-const Navbar = (props) => {
-
+const Navbar = props => {
   // Default Class to apply to Component
-  let classList = `Navbar`;
-
 
   return (
-    <div className={classList}>
-      <div className="navbar-logo">
+    <div className="Navbar">
+      <div className="Navbar__logo">
         <Logo clickable to="/route" />
       </div>
+      <div className="Navbar__main">
+        <div className="Navbar__menu">
+          <div className="Navbar__link-item">
+            <NavLink
+              text="My Directory"
+              to="/"
+              menuList={[
+                { text: "Link 1", to: "/" },
+                { text: "Link 2", to: "/" }
+              ]}
+            />
+            <div className={`Navbar__triangle ${props.menu1}`}></div>
+          </div>
+          <div className="Navbar__link-item">
+            <NavLink
+              text="Community"
+              to="/"
+              menuList={[
+                { text: "Link 1", to: "/" },
+                { text: "Link 2", to: "/" }
+              ]}
+            />
+            <div className={`Navbar__triangle ${props.menu2}`}></div>
+          </div>
+          <div className="Navbar__link-item">
+            <NavLink
+              text="Forum"
+              to="/"
+              menuList={[
+                { text: "Link 1", to: "/" },
+                { text: "Link 2", to: "/" }
+              ]}
+            />
+            <div className={`Navbar__triangle ${props.menu3}`}></div>
+          </div>
+          <div className="Navbar__link-item">
+            <NavLink
+              text="Book A Trip"
+              to="/"
+              menuList={[
+                { text: "Link 1", to: "/" },
+                { text: "Link 2", to: "/" }
+              ]}
+            />
+            <div className={`Navbar__triangle ${props.menu4}`}></div>
+          </div>
+        </div>
+        <div className="Navbar__right">
+          <Icon icon={"search"} size="2x" onClick={props.searchClick} />
+          <Icon icon={["far", "comment-dots"]} size="2x" onClick={props.messageClick} />
+          <ProfilePicture type="extra-small" link="#" image={props.profileImage} />
+        </div>
+      </div>
+      <div className="Navbar__burger">
+        <Burger onClick={props.burgerClick} active={props.showCondensedMenu} />
+        <div className={`Navbar__condensed-menu ${props.condensedMenu}`}>
+          <NavLink text="My Directory" to="/" menuList={[]} />
+          <NavLink text="Community" to="/" menuList={[]} />
+          <NavLink text="Forum" to="/" menuList={[]} />
+          <NavLink text="Book A Trip" to="/" menuList={[]} />
+        </div>
+      </div>
     </div>
   );
-}
+};
+
+Navbar.propTypes = {
+  showCondensedMenu: PropTypes.bool,
+  menu1: PropTypes.string,
+  menu2: PropTypes.string,
+  menu3: PropTypes.string,
+  menu4: PropTypes.string,
```

Comment:
@wchrapcynski Looks like this should be an array!

Reply:
@ZakkMan My thought on this is that only one of these is active at any point. So, there will only be one prop being passed down at any given time. Adding the Navbar__show class makes the triangle visible.

Reply:
This is now an array where the triangle will be active depending on which one has "Navbar__show" in it.